source/feature-renames.json

```diff
@@ -1,4 +1,5 @@
 {
+	"scrollable-code-and-blockquote": "scrollable-areas",
```

Comment:
It would be great to add all previous names in all the rename PRs that came before this rename tracking: https://github.com/refined-github/refined-github/pulls?q=is%3Apr+is%3Aclosed+rename+in%3Atitle+sort%3Acreated-asc

But it's hard to find them, especially because some features have been dropped altogether